src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -58,10 +74,14 @@ export class ItemEditComponent implements OnDestroy, PendingChangesComponent {
   ) {
     this.modeService.mode$.next(Mode.Editing);
     this.subscription = this.fetchState$
-      .pipe(readyData())
+      .pipe(readyData(), map(data => ({
+        ...data.item,
+        durationOn: data.item.duration !== null,
+        enteringTimeMaxOn: !DEFAULT_DATES.includes(data.item.enteringTimeMax),
```

Comment:
why comparing with '1000-01-01T00:00:00Z' and '9999-12-31T23:59:59Z'. Isn't '9999-12-31T23:59:59Z' enough?

Reply:
Fixed